### PR TITLE
Send discussion info to bigquery

### PR DESCRIPTION
**Pull request by hubertdeng123**
This requires webhooks to send first from https://github.com/organizations/getsentry/settings/apps/getsantry/permissions

### CODE REVIEW

File: src/utils/metrics.ts [hunk 1]
```diff
@@ -359,15 +369,26 @@ export async function insertOss(
...
+    data.created_at = comment.created_at;
+    data.updated_at = comment.updated_at;
+    data.target_id = comment.id;
+    data.target_name = discussion.title;
+    data.target_type = 'discussion';
   } else {
     // Unknown payload event, ignoring...
     return {};
   }
-  data.teams = getTeams(
```
Comment (hubertdeng123) [inline src/utils/metrics.ts:366]: I think it's wrong to have a team assigned to every GitHub event, especially if we're only tracking it for `issue` and `issue_comment` event types

File: src/utils/metrics.test.ts [hunk 2]
```diff
@@ -29,202 +29,277 @@ import { insertOss } from './metrics';
...
-      sender: {
-        login: 'username',
-        id: 'user_id',
-      },
-      repository: {
-        full_name: 'getsentry/routing-repo',
-        name: 'routing-repo',
-        owner: {
-          type: 'Organization',
+    describe('lableling events', function () {
```
Comment (azaslavsky) [inline src/utils/metrics.test.ts:32]: nut: spelling

File: src/utils/metrics.test.ts [hunk 3]
```diff
@@ -29,202 +29,277 @@ import { insertOss } from './metrics';
...
-      });
-      await getLabelsTable().insert({
-        label_name: 'Product Area: Other',
-        channel_id: 'CHNLIDRND1',
-        offices: ['sfo'],
+      let dateNowSpy;
+      beforeAll(async () => {
+        dateNowSpy = jest
+          .spyOn(Date, 'now')
+          .mockImplementation(() => 1487076708000);
```
Comment (azaslavsky) [inline src/utils/metrics.test.ts:65]: nit: may be a bit more readable if we use moment to generate this timestamp?

**Review by azaslavsky — APPROVED** [review]

File: src/utils/metrics.test.ts (hunk 2 above)
Comment (hubertdeng123) [reply src/utils/metrics.test.ts]: good catch

File: src/utils/metrics.test.ts (hunk 3 above)
Comment (hubertdeng123) [reply src/utils/metrics.test.ts]: done

**Pull request merged**